numba/targets/arrayobj.py

```diff
@@ -3092,12 +3102,12 @@ def iternext_numpy_getitem(context, builder, sig, args):
     arrcls = context.make_array(arrty)
     arr = arrcls(context, builder, value=flatiter.array)
 
-    res = flatiter.setitem(context, builder, arrty, arr, index, value)
+    _ = flatiter.setitem(context, builder, arrty, arr, index, value)
```

Comment:
assignment can be removed?

Reply:
@esc note the *assignment* can be removed, not the call itself. https://github.com/numba/numba/pull/3927/files#diff-81d8f85ead1fb3ae948c94ed8d2a969eL3095, `setitem` has a side effect